app_dart/test/request_handlers/push_build_status_to_github_test.dart

```diff
@@ -21,201 +18,255 @@ import '../src/datastore/fake_config.dart';
 import '../src/datastore/fake_datastore.dart';
 import '../src/request_handling/api_request_handler_tester.dart';
 import '../src/request_handling/fake_authentication.dart';
-import '../src/request_handling/fake_http.dart';
-import '../src/request_handling/fake_logging.dart';
-import '../src/service/fake_build_status_provider.dart';
 import '../src/service/fake_github_service.dart';
+import '../src/service/fake_scheduler.dart';
 import '../src/utilities/mocks.dart';
 
 void main() {
-  group('PushBuildStatusToGithub', () {
+  group('PushStatusToGithub', () {
```

Comment:
shall we add a test to update status for multiple branches in flutter/flutter?

Reply:
Actually the reason I didn't add tests for different branches is that the more I look at the code the more convinced I am that we don't need tree status for branches.

@christopherfujino do we actually need branch support for tree status?

Reply:
we don't need this.

Reply:
thanks, removed.